scripts/exgfs_wave_post_pnt.sh

```diff
@@ -385,19 +385,13 @@ source "${USHgfs}/preamble.sh"
     if [ "$DOSPC_WAV" = 'YES' ]
     then
       export dtspec=3600.
-      for buoy in $buoys
-      do
-        echo "${USHgfs}/wave_outp_spec.sh $buoy $ymdh spec $SPECDATA > $SPECDATA/spec_$buoy.out 2>&1" >> tmpcmdfile.$FH3
-      done
+      sed "s/^\(.*\)$/${USHgfs//\//\\\/}\/wave_outp_spec.sh \1 ${ymdh} spec ${SPECDATA//\//\\\/} > ${SPECDATA//\//\\\/}\/spec_\1.out 2>&1/" buoy_lst.txt >> "tmpcmdfile.${FH3}"
```

Comment:
Can we do the substitutions ahead of time in bash to avoid the escape hell here?

Reply:
So just as an FYI - the executable this job runs, is going to be completely re-written to actually use MPI instead of having MPI via shell-script.  So I would recommend doing the minimal amount of re-writes to get things working more efficiently while the larger task of completely rewriting this is completed.    The first step towards this is under-review in WW3 now.

Reply:
Thanks for that notice @JessicaMeixner-NOAA!

@WalterKolczynski-NOAA I pre-escaped the paths and verified that the sed commands were the same.